packages/rancher-logging/overlay/templates/loggings/aks/logging.yaml

```diff
@@ -0,0 +1,44 @@
+{{- if and .Values.additionalLoggingSources.aks.enabled }}
+apiVersion: logging.banzaicloud.io/v1beta1
+kind: Logging
+metadata:
+  name: {{ .Release.Name }}-aks
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "logging-operator.labels" . | indent 4 }}
+spec:
+  controlNamespace: {{ .Release.Namespace }}
+  fluentbit:
+    image:
+      repository: {{ template "system_default_registry" . }}{{ .Values.images.fluentbit.repository }}
+      tag: {{ .Values.images.fluentbit.tag }}
+    inputTail:
+      Tag: "aks"
+      Path: "/var/log/azure/kubelet-status.log,/var/log/containers/kube-proxy*.log"
```

Comment:
Does this work? that is really nice if so.

Reply:
It does! It's a comma-seperated list even though it's processed as a string.